.devcontainer/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM mcr.microsoft.com/devcontainers/typescript-node:20-bullseye
+
+# Update the underlying (Debian) OS, to make sure we have the latest security patches and libraries like 'GLIBC' 
+sudo apt-get update  && sudo apt-get -y upgrade
+
+# Update pnpm
+npm install -g pnpm@8.15.7
```

Comment:
I think this use the `RUN` instruction, https://docs.docker.com/reference/dockerfile/#run

I get an error when I start the container:

https://glorious-space-memory-p5x7rj7w99f4qg.github.dev/

```
2024-05-21 19:39:29.343Z: Dockerfile-with-features:62024-05-21 19:39:29.347Z: 
2024-05-21 19:39:29.353Z: --------------------2024-05-21 19:39:29.362Z: 
2024-05-21 19:39:29.369Z:    4 |     
2024-05-21 19:39:29.376Z:    5 |     # Update the underlying (Debian) OS, to make sure we have the latest security patches and libraries like 'GLIBC' 
   6 | >>> sudo apt-get update  && sudo apt-get -y upgrade
   7 |     
   8 |     # Update pnpm
--------------------
2024-05-21 19:39:29.379Z: ERROR: failed to solve: dockerfile parse error on line 6: unknown instruction: sudo
2024-05-21 19:39:29.387Z: Stop: Run: docker buildx build --load --build-arg BUILDKIT_INLINE_CACHE=1 -f /tmp/devcontainercli-root/container-features/0.56.1-1716320365265/Dockerfile-with-features -t vsc-nx-fdb3c16b9558879fc067e7fd4c2a51454587f74e4c9d6b50d5f36ce3dae30086 --target dev_containers_target_stage --build-context dev_containers_feature_content_source=/tmp/devcontainercli-root/container-features/0.56.1-1716320365265 --build-arg _DEV_CONTAINERS_BASE_IMAGE=dev_container_auto_added_stage_label --build-arg _DEV_CONTAINERS_IMAGE_USER=root --build-arg _DEV_CONTAINERS_FEATURE_CONTENT_SOURCE=dev_container_feature_content_temp /var/lib/docker/codespacemount/workspace/nx/.devcontainer
2024-05-21 19:39:29.405Z: Error: Command failed: docker buildx build --load --build-arg BUILDKIT_INLINE_CACHE=1 -f /tmp/devcontainercli-root/container-features/0.56.1-1716320365265/Dockerfile-with-features -t vsc-nx-fdb3c16b9558879fc067e7fd4c2a51454587f74e4c9d6b50d5f36ce3dae30086 --target dev_containers_target_stage --build-context dev_containers_feature_content_source=/tmp/devcontainercli-root/container-features/0.56.1-1716320365265 --build-arg _DEV_CONTAINERS_BASE_IMAGE=dev_container_auto_added_stage_label --build-arg _DEV_CONTAINERS_IMAGE_USER=root --build-arg _DEV_CONTAINERS_FEATURE_CONTENT_SOURCE=dev_container_feature_content_temp /var/lib/docker/codespacemount/workspace/nx/.devcontainer
2024-05-21 19:39:29.407Z:     at j$ (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:465:1933)
2024-05-21 19:39:29.410Z:     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

Reply:
Updated